Monika After Story/game/script-greetings.rpy

```diff
@@ -3324,3 +3324,23 @@ label ch30_reload_delegate:
         call expression reload_label
 
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.greeting_database,
+            eventlabel="greeting_ghost",
+            unlocked=False,
```

Comment:
since unlocked is False here, it wont trigger. Are you planning to unlock this in a future topic?

Reply:
Probably needs to be unlocked by/update scripted for do you like spooks